onnxruntime/core/providers/cpu/tensor/concat.cc

```diff
@@ -63,7 +66,7 @@ Status ConcatBase::PrepareForCompute(OpKernelContext* ctx, int input_count, Prep
 
     // The input_axis_pitch is the number of elements to add to move to the next split axis in the input
     int64_t input_axis_pitch = 1;
-    for (int i = int(data_n.Shape().NumDimensions()); i-- > axis_;)
+    for (int i = int(data_n.Shape().NumDimensions()); i-- > axis;)
```

Comment:
Would static_cast<int>(...) be better than int(...) ?

Reply:
Maybe 'auto i' would be better. Is there a difference in this case?

Reply:
No difference. 
It doesn't conform our coding style:
https://google.github.io/styleguide/cppguide.html#Casting